inst/extdata/install-packages.R

```diff
@@ -252,7 +257,7 @@ setup_esqlabsR <- function() {
   initialize_project()
 
   if (install_option == 1) {
-    renv::snapshot(prompt = FALSE) # snapshot environment
+    renv::snapshot(prompt = FALSE, force = TRUE) # snapshot environment
```

Comment:
What about this? Should we merge this change only, or do you want to integrate it into another PR?